inappmessaging/src/test/java/com/rakuten/tech/mobile/inappmessaging/runtime/view/BaseViewSpec.kt

```diff
@@ -262,7 +262,7 @@ class BaseViewSpec : BaseTest() {
         if (isValid) {
             imageView?.visibility shouldBeEqualTo View.VISIBLE
         } else {
-            imageView?.visibility shouldBeEqualTo View.GONE
+            imageView?.visibility shouldBeEqualTo  8 // View.GONE
```

Comment:
Why change a constant var to a hard-coded number here?

Reply:
Mobscan caught an error for rule Id **android_kotlin_hiddenui** with following description:
"Hidden elements in view can be used to hide data from user. But this data can be leaked"

Reply:
Can this be dismissed as `Used int tests`?

Reply:
Yes, it is possible